docs/control_center_docs/control_center_start_guide.rst

```diff
@@ -0,0 +1,737 @@
+like to use, then click Set.
+
+Using User Generated Default Settings
+-----------------------------------------
+User generated default settings allow you to easily share identical module configurations between multiple modules. For example, if you have parameters 
+that work for your module configuration (gains, modes, baud rate, etc.) that should be set on all of your modules, you would want to use a custom defaults file.
+
+.. note::
+    User generated Defaults files include the Module ID parameter. This means that setting up modules with custom Default files will result in collisions on 
+    a serial bus, and it is recommended that you set defaults with only one module connected at a time, and if desired, assign unique Module IDs 
+    after setting all other parameters to connect via a bus.
+
+Creating a User Defaults File
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The first step in creating a defaults file is finding the parameters that work best for your modules. Once you have a set of parameters 
+(across the general, tuning, and advanced tabs) that work well, you can continue. 
+
+1. With your module connected, navigate to the general tab and find the MODULE PRE-SETTINGS
+2. Find the Export button on the right side
+3. Click Export, and you will see a File Explorer window open
+4. Here, choose a name and location for your generated defaults file. In this example I will save the file as “example_defaults.json”
+5. You will then be given the option to import the file directly into the Control Center. In order to use this file to configure other modules, you must import it into the Control Center
+
+.. image:: ../_static/control_center_pics/import_defaults_popup.png
+
+6. After clicking Yes, you will see your imported defaults as an option in the dropdown. This file will now always be available in the Control Center
+7. Once in the Control Center, you can use the Set button exactly as you would with a Vertiq supplied Defaults file
+
+Importing a User Defaults File
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+It is also possible to import externally generated Defaults files into the Control Center. To do so:
+
+#. Select the Import option
+#. Use the File Explorer to find the Defaults file you would like to import
+#. You will see the file appear in the dropdown of other Defaults files, and it is ready for use
+
+******************************************
+Testing Modules with the Control Center
+******************************************
+The Control Center provides a basic suite of testing options to ensure that your module is functioning properly. These options are available in the testing tab, 
+available on the left side.
+
+.. note::
+    Before using the Control Center's testing functionality, we recommend that you increase your module's timeout parameter to 1.5s. 
+    This parameter is available through the tuning tab. If you do not increase the timeout value, your module may fail to spin as expected.
```

Comment:
You may want to mention that people might want to consider setting it back to a lower value when they are done depending on what they really want the timeout to be.

Also may be worth explaining that the reason for this is because the Control Center only heartbeats once a second, so if your timeout is less than 1s your module will timeout